app/js/cartogram.js

```diff
@@ -93,7 +90,7 @@
       var i = 0;
       while (i++ < iterations) {
         var areas = objects.map(path.area);
-        console.log(objects)
+        //console.log(objects)
```

Comment:
also, we should be removing all console.logs from the project, and using the debugger for this instead/just temporarily putting in console logs